src/view/com/composer/Composer.tsx

```diff
@@ -1261,9 +1379,14 @@ const styles = StyleSheet.create({
     justifyContent: 'center',
     marginRight: 5,
   },
+  post: {
+    marginHorizontal: 16,
+  },
+  inactivePost: {
+    opacity: 0.5,
+  },
   scrollView: {
     flex: 1,
-    paddingHorizontal: 16,
```

Comment:
Replacing this with margins on inner content so we can let the footer go full bleed even when rendered within the post list.